HISTORY.md

```diff
@@ -1,9 +1,11 @@
-unreleased
-==========
+1.5.0 / 2023-04-10
```

Comment:
Not sure if this should go under "unreleased" & if the changes for `cookie@0.4.2` which were under unreleased should be kept.

Never changed a `HISTORY.md`, please let me know in case I've done this incorrectly here or please point me to an appropriate guideline (if any) to follow in this.

I think the current bump should be a `minor` as per semver, hence upgraded from `1.4.6 -> 1.5.0`

Reply:
It's no problem. This should be kept as `unreleased` in the pull request, since the charges haven't been released at the time of the pr merge. During the release, the banner gets changed to the version published as well as the date 👍

Reply:
Updated this to `unreleased` & appended to the notes that already existed within unreleased